app/assets/javascripts/admin/developers.coffee

```diff
@@ -0,0 +1,7 @@
+$('#admin_developer_active').click ->
+	$('#date_leave_project').toggle()
```

Comment:
This code isn't necessary. Look, you do this on events.coffee. Remove this file.